File: Sources/PDFjet/Color.swift
/**
 * Color.swift
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */

/**
 * Used to specify the pen and brush colors.
 * @see <a href="http://www.w3.org/TR/css3-color/#svg-color">http://www.w3.org/TR/css3-color/#svg-color</a>
 */
public class Color {
    /// The color named "transparent"
    public static let transparent: Int32 = -1

    /// The color named "aliceblue"
    public static let aliceblue: Int32 = 0xf0f8ff
    /// The color named "antiquewhite"
    public static let antiquewhite: Int32 = 0xfaebd7
    /// The color named "aqua"
    public static let aqua: Int32 = 0x00ffff
    /// The color named "aquamarine"
    public static let aquamarine: Int32 = 0x7fffd4
    /// The color named "azure"
    public static let azure: Int32 = 0xf0ffff
    /// The color named "beige"
    public static let beige: Int32 = 0xf5f5dc
    /// The color named "bisque"
    public static let bisque: Int32 = 0xffe4c4
    /// The color named "black"
    public static let black: Int32 = 0x000000
    /// The color named "blanchedalmond"
    public static let blanchedalmond: Int32 = 0xffebcd
    /// The color named "blue"
    public static let blue: Int32 = 0x0000ff
    /// The color named "blueviolet"
    public static let blueviolet: Int32 = 0x8a2be2
    /// The color named "brown"
    public static let brown: Int32 = 0xa52a2a
    /// The color named "burlywood"
    public static let burlywood: Int32 = 0xdeb887
    /// The color named "cadetblue"
    public static let cadetblue: Int32 = 0x5f9ea0
    /// The color named "chartreuse"
    public static let chartreuse: Int32 = 0x7fff00
    /// The color named "chocolate"
    public static let chocolate: Int32 = 0xd2691e
    /// The color named "coral"
    public static let coral: Int32 = 0xff7f50
    /// The color named "cornflowerblue"
    public static let cornflowerblue: Int32 = 0x6495ed
    /// The color named "cornsilk"
    public static let cornsilk: Int32 = 0xfff8dc
    /// The color named "crimson"
    public static let crimson: Int32 = 0xdc143c
    /// The color named "cyan"
    public static let cyan: Int32 = 0x00ffff
    /// The color named "darkblue"
    public static let darkblue: Int32 = 0x00008b
    /// The color named "darkcyan"
    public static let darkcyan: Int32 = 0x008b8b
    /// The color named "darkgoldenrod"
    public static let darkgoldenrod: Int32 = 0xb8860b
    /// The color named "darkgray"
    public static let darkgray: Int32 = 0xa9a9a9
    /// The color named "darkgreen"
    public static let darkgreen: Int32 = 0x006400
    /// The color named "darkgrey"
    public static let darkgrey: Int32 = 0xa9a9a9
    /// The color named "darkkhaki"
    public static let darkkhaki: Int32 = 0xbdb76b
    /// The color named "darkmagenta"
    public static let darkmagenta: Int32 = 0x8b008b
    /// The color named "darkolivegreen"
    public static let darkolivegreen: Int32 = 0x556b2f
    /// The color named "darkorange"
    public static let darkorange: Int32 = 0xff8c00
    /// The color named "darkorchid"
    public static let darkorchid: Int32 = 0x9932cc
    /// The color named "darkred"
    public static let darkred: Int32 = 0x8b0000
    /// The color named "darksalmon"
    public static let darksalmon: Int32 = 0xe9967a
    /// The color named "darkseagreen"
    public static let darkseagreen: Int32 = 0x8fbc8f
    /// The color named "darkslateblue"
    public static let darkslateblue: Int32 = 0x483d8b
    /// The color named "darkslategray"
    public static let darkslategray: Int32 = 0x2f4f4f
    /// The color named "darkslategrey"
    public static let darkslategrey: Int32 = 0x2f4f4f
    /// The color named "darkturquoise"
    public static let darkturquoise: Int32 = 0x00ced1
    /// The color named "darkviolet"
    public static let darkviolet: Int32 = 0x9400d3
    /// The color named "deeppink"
    public static let deeppink: Int32 = 0xff1493
    /// The color named "deepskyblue"
    public static let deepskyblue: Int32 = 0x00bfff
    /// The color named "dimgray"
    public static let dimgray: Int32 = 0x696969
    /// The color named "dimgrey"
    public static let dimgrey: Int32 = 0x696969
    /// The color named "dodgerblue"
    public static let dodgerblue: Int32 = 0x1e90ff
    /// The color named "firebrick"
    public static let firebrick: Int32 = 0xb22222
    /// The color named "floralwhite"
    public static let floralwhite: Int32 = 0xfffaf0
    /// The color named "forestgreen"
    public static let forestgreen: Int32 = 0x228b22
    /// The color named "fuchsia"
    public static let fuchsia: Int32 = 0xff00ff
    /// The color named "gainsboro"
    public static let gainsboro: Int32 = 0xdcdcdc
    /// The color named "ghostwhite"
    public static let ghostwhite: Int32 = 0xf8f8ff
    /// The color named "gold"
    public static let gold: Int32 = 0xffd700
    /// The color named "goldenrod"
    public static let goldenrod: Int32 = 0xdaa520
    /// The color named "gray"
    public static let gray: Int32 = 0x808080
    /// The color named "green"
    public static let green: Int32 = 0x008000
    /// The color named "greenyellow"
    public static let greenyellow: Int32 = 0xadff2f
    /// The color named "grey"
    public static let grey: Int32 = 0x808080
    /// The color named "honeydew"
    public static let honeydew: Int32 = 0xf0fff0
    /// The color named "hotpink"
    public static let hotpink: Int32 = 0xff69b4
    /// The color named "indianred"
    public static let indianred: Int32 = 0xcd5c5c
    /// The color named "indigo"
    public static let indigo: Int32 = 0x4b0082
    /// The color named "ivory"
    public static let ivory: Int32 = 0xfffff0
    /// The color named "khaki"
    public static let khaki: Int32 = 0xf0e68c
    /// The color named "lavender"
    public static let lavender: Int32 = 0xe6e6fa
    /// The color named "lavenderblush"
    public static let lavenderblush: Int32 = 0xfff0f5
    /// The color named "lawngreen"
    public static let lawngreen: Int32 = 0x7cfc00
    /// The color named "lemonchiffon"
    public static let lemonchiffon: Int32 = 0xfffacd
    /// The color named "lightblue"
    public static let lightblue: Int32 = 0xadd8e6
    /// The color named "lightcoral"
    public static let lightcoral: Int32 = 0xf08080
    /// The color named "lightcyan"
    public static let lightcyan: Int32 = 0xe0ffff
    /// The color named "lightgoldenrodyellow"
    public static let lightgoldenrodyellow: Int32 = 0xfafad2
    /// The color named "lightgray"
    public static let lightgray: Int32 = 0xd3d3d3
    /// The color named "lightgreen"
    public static let lightgreen: Int32 = 0x90ee90
    /// The color named "lightgrey"
    public static let lightgrey: Int32 = 0xd3d3d3
    /// The color named "lightpink"
    public static let lightpink: Int32 = 0xffb6c1
    /// The color named "lightsalmon"
    public static let lightsalmon: Int32 = 0xffa07a
    /// The color named "lightseagreen"
    public static let lightseagreen: Int32 = 0x20b2aa
    /// The color named "lightskyblue"
    public static let lightskyblue: Int32 = 0x87cefa
    /// The color named "lightslategray"
    public static let lightslategray: Int32 = 0x778899
    /// The color named "lightslategrey"
    public static let lightslategrey: Int32 = 0x778899
    /// The color named "lightsteelblue"
    public static let lightsteelblue: Int32 = 0xb0c4de
    /// The color named "lightyellow"
    public static let lightyellow: Int32 = 0xffffe0
    /// The color named "lime"
    public static let lime: Int32 = 0x00ff00
    /// The color named "limegreen"
    public static let limegreen: Int32 = 0x32cd32
    /// The color named "linen"
    public static let linen: Int32 = 0xfaf0e6
    /// The color named "magenta"
    public static let magenta: Int32 = 0xff00ff
    /// The color named "maroon"
    public static let maroon: Int32 = 0x800000
    /// The color named "mediumaquamarine"
    public static let mediumaquamarine: Int32 = 0x66cdaa
    /// The color named "mediumblue"
    public static let mediumblue: Int32 = 0x0000cd
    /// The color named "mediumorchid"
    public static let mediumorchid: Int32 = 0xba55d3
    /// The color named "mediumpurple"
    public static let mediumpurple: Int32 = 0x9370db
    /// The color named "mediumseagreen"
    public static let mediumseagreen: Int32 = 0x3cb371
    /// The color named "mediumslateblue"
    public static let mediumslateblue: Int32 = 0x7b68ee
    /// The color named "mediumspringgreen"
    public static let mediumspringgreen: Int32 = 0x00fa9a
    /// The color named "mediumturquoise"
    public static let mediumturquoise: Int32 = 0x48d1cc
    /// The color named "mediumvioletred"
    public static let mediumvioletred: Int32 = 0xc71585
    /// The color named "midnightblue"
    public static let midnightblue: Int32 = 0x191970
    /// The color named "mintcream"
    public static let mintcream: Int32 = 0xf5fffa
    /// The color named "mistyrose"
    public static let mistyrose: Int32 = 0xffe4e1
    /// The color named "moccasin"
    public static let moccasin: Int32 = 0xffe4b5
    /// The color named "navajowhite"
    public static let navajowhite: Int32 = 0xffdead
    /// The color named "navy"
    public static let navy: Int32 = 0x000080
    /// The color named "oldlace"
    public static let oldlace: Int32 = 0xfdf5e6
    /// The color named "olive"
    public static let olive: Int32 = 0x808000
    /// The color named "olivedrab"
    public static let olivedrab: Int32 = 0x6b8e23
    /// The color named "orange"
    public static let orange: Int32 = 0xffa500
    /// The color named "orangered"
    public static let orangered: Int32 = 0xff4500
    /// The color named "orchid"
    public static let orchid: Int32 = 0xda70d6
    /// The color named "palegoldenrod"
    public static let palegoldenrod: Int32 = 0xeee8aa
    /// The color named "palegreen"
    public static let palegreen: Int32 = 0x98fb98
    /// The color named "paleturquoise"
    public static let paleturquoise: Int32 = 0xafeeee
    /// The color named "palevioletred"
    public static let palevioletred: Int32 = 0xdb7093
    /// The color named "papayawhip"
    public static let papayawhip: Int32 = 0xffefd5
    /// The color named "peachpuff"
    public static let peachpuff: Int32 = 0xffdab9
    /// The color named "peru"
    public static let peru: Int32 = 0xcd853f
    /// The color named "pink"
    public static let pink: Int32 = 0xffc0cb
    /// The color named "plum"
    public static let plum: Int32 = 0xdda0dd
    /// The color named "powderblue"
    public static let powderblue: Int32 = 0xb0e0e6
    /// The color named "purple"
    public static let purple: Int32 = 0x800080
    /// The color named "red"
    public static let red: Int32 = 0xff0000
    /// The color named "rosybrown"
    public static let rosybrown: Int32 = 0xbc8f8f
    /// The color named "royalblue"
    public static let royalblue: Int32 = 0x4169e1
    /// The color named "saddlebrown"
    public static let saddlebrown: Int32 = 0x8b4513
    /// The color named "salmon"
    public static let salmon: Int32 = 0xfa8072
    /// The color named "sandybrown"
    public static let sandybrown: Int32 = 0xf4a460
    /// The color named "seagreen"
    public static let seagreen: Int32 = 0x2e8b57
    /// The color named "seashell"
    public static let seashell: Int32 = 0xfff5ee
    /// The color named "sienna"
    public static let sienna: Int32 = 0xa0522d
    /// The color named "silver"
    public static let silver: Int32 = 0xc0c0c0
    /// The color named "skyblue"
    public static let skyblue: Int32 = 0x87ceeb
    /// The color named "slateblue"
    public static let slateblue: Int32 = 0x6a5acd
    /// The color named "slategray"
    public static let slategray: Int32 = 0x708090
    /// The color named "slategrey"
    public static let slategrey: Int32 = 0x708090
    /// The color named "snow"
    public static let snow: Int32 = 0xfffafa
    /// The color named "springgreen"
    public static let springgreen: Int32 = 0x00ff7f
    /// The color named "steelblue"
    public static let steelblue: Int32 = 0x4682b4
    /// The color named "tan"
    public static let tan: Int32 = 0xd2b48c
    /// The color named "teal"
    public static let teal: Int32 = 0x008080
    /// The color named "thistle"
    public static let thistle: Int32 = 0xd8bfd8
    /// The color named "tomato"
    public static let tomato: Int32 = 0xff6347
    /// The color named "turquoise"
    public static let turquoise: Int32 = 0x40e0d0
    /// The color named "violet"
    public static let violet: Int32 = 0xee82ee
    /// The color named "wheat"
    public static let wheat: Int32 = 0xf5deb3
    /// The color named "white"
    public static let white: Int32 = 0xffffff
    /// The color named "whitesmoke"
    public static let whitesmoke: Int32 = 0xf5f5f5
    /// The color named "yellow"
    public static let yellow: Int32 = 0xffff00
    /// The color named "yellowgreen"
    public static let yellowgreen: Int32 = 0x9acd32

    /// The color named "oldgloryred"
    public static let oldgloryred: Int32 = 0xb22234
    /// The color named "oldgloryblue"
    public static let oldgloryblue: Int32 = 0x3c3b6e
}

File: net/pdfjet/Color.cs
/*
 * Color.cs
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */

namespace PDFjet.NET {
/// <summary>
/// Used to specify the pen and brush colors.
/// </summary>
/// <seealso href="http://www.w3.org/TR/css3-color/#svg-color">http://www.w3.org/TR/css3-color/#svg-color</seealso>
public class Color {
    /// <summary>The color named "transparent"</summary>
    public const int transparent = -1;

    /// <summary>The color named "aliceblue"</summary>
    public const int aliceblue = 0xf0f8ff;
    /// <summary>The color named "antiquewhite"</summary>
    public const int antiquewhite = 0xfaebd7;
    /// <summary>The color named "aqua"</summary>
    public const int aqua = 0x00ffff;
    /// <summary>The color named "aquamarine"</summary>
    public const int aquamarine = 0x7fffd4;
    /// <summary>The color named "azure"</summary>
    public const int azure = 0xf0ffff;
    /// <summary>The color named "beige"</summary>
    public const int beige = 0xf5f5dc;
    /// <summary>The color named "bisque"</summary>
    public const int bisque = 0xffe4c4;
    /// <summary>The color named "black"</summary>
    public const int black = 0x000000;
    /// <summary>The color named "blanchedalmond"</summary>
    public const int blanchedalmond = 0xffebcd;
    /// <summary>The color named "blue"</summary>
    public const int blue = 0x0000ff;
    /// <summary>The color named "blueviolet"</summary>
    public const int blueviolet = 0x8a2be2;
    /// <summary>The color named "brown"</summary>
    public const int brown = 0xa52a2a;
    /// <summary>The color named "burlywood"</summary>
    public const int burlywood = 0xdeb887;
    /// <summary>The color named "cadetblue"</summary>
    public const int cadetblue = 0x5f9ea0;
    /// <summary>The color named "chartreuse"</summary>
    public const int chartreuse = 0x7fff00;
    /// <summary>The color named "chocolate"</summary>
    public const int chocolate = 0xd2691e;
    /// <summary>The color named "coral"</summary>
    public const int coral = 0xff7f50;
    /// <summary>The color named "cornflowerblue"</summary>
    public const int cornflowerblue = 0x6495ed;
    /// <summary>The color named "cornsilk"</summary>
    public const int cornsilk = 0xfff8dc;
    /// <summary>The color named "crimson"</summary>
    public const int crimson = 0xdc143c;
    /// <summary>The color named "cyan"</summary>
    public const int cyan = 0x00ffff;
    /// <summary>The color named "darkblue"</summary>
    public const int darkblue = 0x00008b;
    /// <summary>The color named "darkcyan"</summary>
    public const int darkcyan = 0x008b8b;
    /// <summary>The color named "darkgoldenrod"</summary>
    public const int darkgoldenrod = 0xb8860b;
    /// <summary>The color named "darkgray"</summary>
    public const int darkgray = 0xa9a9a9;
    /// <summary>The color named "darkgreen"</summary>
    public const int darkgreen = 0x006400;
    /// <summary>The color named "darkgrey"</summary>
    public const int darkgrey = 0xa9a9a9;
    /// <summary>The color named "darkkhaki"</summary>
    public const int darkkhaki = 0xbdb76b;
    /// <summary>The color named "darkmagenta"</summary>
    public const int darkmagenta = 0x8b008b;
    /// <summary>The color named "darkolivegreen"</summary>
    public const int darkolivegreen = 0x556b2f;
    /// <summary>The color named "darkorange"</summary>
    public const int darkorange = 0xff8c00;
    /// <summary>The color named "darkorchid"</summary>
    public const int darkorchid = 0x9932cc;
    /// <summary>The color named "darkred"</summary>
    public const int darkred = 0x8b0000;
    /// <summary>The color named "darksalmon"</summary>
    public const int darksalmon = 0xe9967a;
    /// <summary>The color named "darkseagreen"</summary>
    public const int darkseagreen = 0x8fbc8f;
    /// <summary>The color named "darkslateblue"</summary>
    public const int darkslateblue = 0x483d8b;
    /// <summary>The color named "darkslategray"</summary>
    public const int darkslategray = 0x2f4f4f;
    /// <summary>The color named "darkslategrey"</summary>
    public const int darkslategrey = 0x2f4f4f;
    /// <summary>The color named "darkturquoise"</summary>
    public const int darkturquoise = 0x00ced1;
    /// <summary>The color named "darkviolet"</summary>
    public const int darkviolet = 0x9400d3;
    /// <summary>The color named "deeppink"</summary>
    public const int deeppink = 0xff1493;
    /// <summary>The color named "deepskyblue"</summary>
    public const int deepskyblue = 0x00bfff;
    /// <summary>The color named "dimgray"</summary>
    public const int dimgray = 0x696969;
    /// <summary>The color named "dimgrey"</summary>
    public const int dimgrey = 0x696969;
    /// <summary>The color named "dodgerblue"</summary>
    public const int dodgerblue = 0x1e90ff;
    /// <summary>The color named "firebrick"</summary>
    public const int firebrick = 0xb22222;
    /// <summary>The color named "floralwhite"</summary>
    public const int floralwhite = 0xfffaf0;
    /// <summary>The color named "forestgreen"</summary>
    public const int forestgreen = 0x228b22;
    /// <summary>The color named "fuchsia"</summary>
    public const int fuchsia = 0xff00ff;
    /// <summary>The color named "gainsboro"</summary>
    public const int gainsboro = 0xdcdcdc;
    /// <summary>The color named "ghostwhite"</summary>
    public const int ghostwhite = 0xf8f8ff;
    /// <summary>The color named "gold"</summary>
    public const int gold = 0xffd700;
    /// <summary>The color named "goldenrod"</summary>
    public const int goldenrod = 0xdaa520;
    /// <summary>The color named "gray"</summary>
    public const int gray = 0x808080;
    /// <summary>The color named "green"</summary>
    public const int green = 0x008000;
    /// <summary>The color named "greenyellow"</summary>
    public const int greenyellow = 0xadff2f;
    /// <summary>The color named "grey"</summary>
    public const int grey = 0x808080;
    /// <summary>The color named "honeydew"</summary>
    public const int honeydew = 0xf0fff0;
    /// <summary>The color named "hotpink"</summary>
    public const int hotpink = 0xff69b4;
    /// <summary>The color named "indianred"</summary>
    public const int indianred = 0xcd5c5c;
    /// <summary>The color named "indigo"</summary>
    public const int indigo = 0x4b0082;
    /// <summary>The color named "ivory"</summary>
    public const int ivory = 0xfffff0;
    /// <summary>The color named "khaki"</summary>
    public const int khaki = 0xf0e68c;
    /// <summary>The color named "lavender"</summary>
    public const int lavender = 0xe6e6fa;
    /// <summary>The color named "lavenderblush"</summary>
    public const int lavenderblush = 0xfff0f5;
    /// <summary>The color named "lawngreen"</summary>
    public const int lawngreen = 0x7cfc00;
    /// <summary>The color named "lemonchiffon"</summary>
    public const int lemonchiffon = 0xfffacd;
    /// <summary>The color named "lightblue"</summary>
    public const int lightblue = 0xadd8e6;
    /// <summary>The color named "lightcoral"</summary>
    public const int lightcoral = 0xf08080;
    /// <summary>The color named "lightcyan"</summary>
    public const int lightcyan = 0xe0ffff;
    /// <summary>The color named "lightgoldenrodyellow"</summary>
    public const int lightgoldenrodyellow = 0xfafad2;
    /// <summary>The color named "lightgray"</summary>
    public const int lightgray = 0xd3d3d3;
    /// <summary>The color named "lightgreen"</summary>
    public const int lightgreen = 0x90ee90;
    /// <summary>The color named "lightgrey"</summary>
    public const int lightgrey = 0xd3d3d3;
    /// <summary>The color named "lightpink"</summary>
    public const int lightpink = 0xffb6c1;
    /// <summary>The color named "lightsalmon"</summary>
    public const int lightsalmon = 0xffa07a;
    /// <summary>The color named "lightseagreen"</summary>
    public const int lightseagreen = 0x20b2aa;
    /// <summary>The color named "lightskyblue"</summary>
    public const int lightskyblue = 0x87cefa;
    /// <summary>The color named "lightslategray"</summary>
    public const int lightslategray = 0x778899;
    /// <summary>The color named "lightslategrey"</summary>
    public const int lightslategrey = 0x778899;
    /// <summary>The color named "lightsteelblue"</summary>
    public const int lightsteelblue = 0xb0c4de;
    /// <summary>The color named "lightyellow"</summary>
    public const int lightyellow = 0xffffe0;
    /// <summary>The color named "lime"</summary>
    public const int lime = 0x00ff00;
    /// <summary>The color named "limegreen"</summary>
    public const int limegreen = 0x32cd32;
    /// <summary>The color named "linen"</summary>
    public const int linen = 0xfaf0e6;
    /// <summary>The color named "magenta"</summary>
    public const int magenta = 0xff00ff;
    /// <summary>The color named "maroon"</summary>
    public const int maroon = 0x800000;
    /// <summary>The color named "mediumaquamarine"</summary>
    public const int mediumaquamarine = 0x66cdaa;
    /// <summary>The color named "mediumblue"</summary>
    public const int mediumblue = 0x0000cd;
    /// <summary>The color named "mediumorchid"</summary>
    public const int mediumorchid = 0xba55d3;
    /// <summary>The color named "mediumpurple"</summary>
    public const int mediumpurple = 0x9370db;
    /// <summary>The color named "mediumseagreen"</summary>
    public const int mediumseagreen = 0x3cb371;
    /// <summary>The color named "mediumslateblue"</summary>
    public const int mediumslateblue = 0x7b68ee;
    /// <summary>The color named "mediumspringgreen"</summary>
    public const int mediumspringgreen = 0x00fa9a;
    /// <summary>The color named "mediumturquoise"</summary>
    public const int mediumturquoise = 0x48d1cc;
    /// <summary>The color named "mediumvioletred"</summary>
    public const int mediumvioletred = 0xc71585;
    /// <summary>The color named "midnightblue"</summary>
    public const int midnightblue = 0x191970;
    /// <summary>The color named "mintcream"</summary>
    public const int mintcream = 0xf5fffa;
    /// <summary>The color named "mistyrose"</summary>
    public const int mistyrose = 0xffe4e1;
    /// <summary>The color named "moccasin"</summary>
    public const int moccasin = 0xffe4b5;
    /// <summary>The color named "navajowhite"</summary>
    public const int navajowhite = 0xffdead;
    /// <summary>The color named "navy"</summary>
    public const int navy = 0x000080;
    /// <summary>The color named "oldlace"</summary>
    public const int oldlace = 0xfdf5e6;
    /// <summary>The color named "olive"</summary>
    public const int olive = 0x808000;
    /// <summary>The color named "olivedrab"</summary>
    public const int olivedrab = 0x6b8e23;
    /// <summary>The color named "orange"</summary>
    public const int orange = 0xffa500;
    /// <summary>The color named "orangered"</summary>
    public const int orangered = 0xff4500;
    /// <summary>The color named "orchid"</summary>
    public const int orchid = 0xda70d6;
    /// <summary>The color named "palegoldenrod"</summary>
    public const int palegoldenrod = 0xeee8aa;
    /// <summary>The color named "palegreen"</summary>
    public const int palegreen = 0x98fb98;
    /// <summary>The color named "paleturquoise"</summary>
    public const int paleturquoise = 0xafeeee;
    /// <summary>The color named "palevioletred"</summary>
    public const int palevioletred = 0xdb7093;
    /// <summary>The color named "papayawhip"</summary>
    public const int papayawhip = 0xffefd5;
    /// <summary>The color named "peachpuff"</summary>
    public const int peachpuff = 0xffdab9;
    /// <summary>The color named "peru"</summary>
    public const int peru = 0xcd853f;
    /// <summary>The color named "pink"</summary>
    public const int pink = 0xffc0cb;
    /// <summary>The color named "plum"</summary>
    public const int plum = 0xdda0dd;
    /// <summary>The color named "powderblue"</summary>
    public const int powderblue = 0xb0e0e6;
    /// <summary>The color named "purple"</summary>
    public const int purple = 0x800080;
    /// <summary>The color named "red"</summary>
    public const int red = 0xff0000;
    /// <summary>The color named "rosybrown"</summary>
    public const int rosybrown = 0xbc8f8f;
    /// <summary>The color named "royalblue"</summary>
    public const int royalblue = 0x4169e1;
    /// <summary>The color named "saddlebrown"</summary>
    public const int saddlebrown = 0x8b4513;
    /// <summary>The color named "salmon"</summary>
    public const int salmon = 0xfa8072;
    /// <summary>The color named "sandybrown"</summary>
    public const int sandybrown = 0xf4a460;
    /// <summary>The color named "seagreen"</summary>
    public const int seagreen = 0x2e8b57;
    /// <summary>The color named "seashell"</summary>
    public const int seashell = 0xfff5ee;
    /// <summary>The color named "sienna"</summary>
    public const int sienna = 0xa0522d;
    /// <summary>The color named "silver"</summary>
    public const int silver = 0xc0c0c0;
    /// <summary>The color named "skyblue"</summary>
    public const int skyblue = 0x87ceeb;
    /// <summary>The color named "slateblue"</summary>
    public const int slateblue = 0x6a5acd;
    /// <summary>The color named "slategray"</summary>
    public const int slategray = 0x708090;
    /// <summary>The color named "slategrey"</summary>
    public const int slategrey = 0x708090;
    /// <summary>The color named "snow"</summary>
    public const int snow = 0xfffafa;
    /// <summary>The color named "springgreen"</summary>
    public const int springgreen = 0x00ff7f;
    /// <summary>The color named "steelblue"</summary>
    public const int steelblue = 0x4682b4;
    /// <summary>The color named "tan"</summary>
    public const int tan = 0xd2b48c;
    /// <summary>The color named "teal"</summary>
    public const int teal = 0x008080;
    /// <summary>The color named "thistle"</summary>
    public const int thistle = 0xd8bfd8;
    /// <summary>The color named "tomato"</summary>
    public const int tomato = 0xff6347;
    /// <summary>The color named "turquoise"</summary>
    public const int turquoise = 0x40e0d0;
    /// <summary>The color named "violet"</summary>
    public const int violet = 0xee82ee;
    /// <summary>The color named "wheat"</summary>
    public const int wheat = 0xf5deb3;
    /// <summary>The color named "white"</summary>
    public const int white = 0xffffff;
    /// <summary>The color named "whitesmoke"</summary>
    public const int whitesmoke = 0xf5f5f5;
    /// <summary>The color named "yellow"</summary>
    public const int yellow = 0xffff00;
    /// <summary>The color named "yellowgreen"</summary>
    public const int yellowgreen = 0x9acd32;

    /// <summary>The color named "oldgloryred"</summary>
    public const int oldgloryred = 0xb22234;
    /// <summary>The color named "oldgloryblue"</summary>
    public const int oldgloryblue = 0x3c3b6e;
}
}    // End of namespace PDFjet.NET
